specifyweb/urls.py

```diff
@@ -20,7 +20,8 @@
 from .workbench import urls as wb_urls
 
 urlpatterns = [
-    url(r'^favicon.ico', RedirectView.as_view(url='/static/img/fav_icon.png')),
+    # First draft, need to see how this is handled
+    url(r'^favicon.ico', RedirectView.as_view(url='/static/img/short_logo.svg')), 
```

Comment:
This just tells to redirect requests to /favicon.ico` to `/static/img/short_logo.svg`.

Need to test if browsers can handle favicon.ico being an SVG. If they can't, you would have to use a PNG here instead

Reply:
The last 3 major releases of Chrome, Edge, Firefox, and Opera support this.

https://caniuse.com/link-icon-svg

Safari (of course) does not support this, but there is a special icon just for that browser ( see `safari-pinned-tab.svg`)

<img width="122" alt="image" src="https://user-images.githubusercontent.com/37256050/210698720-2b9790eb-a5a8-496a-9ccf-aa8fc1c90849.png">

This introduces some odd behavior, since it must be a silouette. What do you think?